pkg/config/linters_settings.go

```diff
@@ -29,6 +29,10 @@ var defaultLintersSettings = LintersSettings{
 	Godox: GodoxSettings{
 		Keywords: []string{},
 	},
+	Godot: GodotSettings{
+		Scope:  "declarations",
+		Period: true,
```

Comment:
In `.golangci.example.yml` `capital` is set to true but since it's lacking in these settings it will bey default be set to `false`. Wouldn't it make more sense to have the example configuration represent the same as the default configuration? At least that's how it's usually configured I think?

Reply:
yes, the values in `.golangci.example.yml` must not be the default values. The default values are in the comments.

Reply:
Got it. Will change once get to laptop.

Reply:
@ldez @bombsimon changes applied, will be glad if you can check it again.